### DISCUSSION TITLE
Codes with zero utilization

### DISCUSSION THREAD

**Discussion by analyst116** [CMSgov/hospital-price-transparency#141] [Q&A] [answered]
Can codes with 0 utilization within the last year be omitted from the MRF?

**Comment by RidenourK** [comment, marked as answer]
@analyst116. Thank you for your question. 

The comprehensive machine-readable file (MRF) should include the standard charges the hospital has established for **all** corresponding items and services provided by the hospital. Items and services as defined at § 180.20 means **all** items and services, including individual items and services and service packages, that could be provided by a hospital to a patient in connection with an inpatient admission or an outpatient department visit for which the hospital has established a standard charge. Examples of hospital items and services include supplies and procedures, room and board, use of the facility and other items (generally described as facility fees), services of employed physicians and non-physician practitioners (generally reflected as professional charges), and any other items or services for which a hospital has established a standard charge.
